src/Stack/Options.hs

```diff
@@ -294,6 +303,15 @@ configOptsParser hide0 =
             hide
   where hide = hideMods (hide0 /= OuterGlobalOpts)
 
+readAbsDir :: ReadM (Path Abs Dir)
```

Comment:
I can buy that this is needed for good errors.  However, `parseAbsDir` directly will also throw a not-so-bad looking error